.appveyor.yml

```diff
@@ -7,36 +7,59 @@ clone_depth: 5
 environment:
   APPVEYOR_SAVE_CACHE_ON_ERROR: true
   CLCACHE_SERVER: 1
-  PACKAGES: berkeleydb boost-filesystem boost-signals2 boost-test libevent openssl rapidcheck zeromq
+  PACKAGES: berkeleydb boost-filesystem boost-signals2 boost-test libevent openssl rapidcheck zeromq protobuf double-conversion
   PATH: 'C:\Python37-x64;C:\Python37-x64\Scripts;%PATH%'
   PYTHONUTF8: 1
+  QT_DOWNLOAD_URL: 'https://github.com/sipsorcery/qt_win_binary/releases/download/v1.0/Qt5.9.7_ssl_x64_static_vs2017.zip'
+  QT_DOWNLOAD_HASH: 'D4D35B8112302B67E5610A03421BB3E43FE13F14D9A5F637C22AE60DCEC0E0F5'
+  QT_LOCAL_PATH: 'C:\Qt5.9.7_ssl_x64_static_vs2017'
+  LIBPROTOBUFLITE_PATH: 'c:\tools\vcpkg\installed\x64-windows-static\lib\libprotobuf-lite.lib'
+  LIBPROTOBUFLITE_DESTINATION_PATH: 'c:\tools\vcpkg\installed\x64-windows-static\lib\libprotobuf-lite.lib'
 cache:
-- C:\tools\vcpkg\installed -> .appveyor.yml
+- C:\tools\vcpkg\installed
 - C:\Users\appveyor\clcache -> .appveyor.yml, build_msvc\**, **\Makefile.am, **\*.vcxproj.in
+- C:\Qt5.9.7_ssl_x64_static_vs2017
 install:
 - cmd: pip install --quiet git+https://github.com/frerich/clcache.git@v4.2.0
 # Disable zmq test for now since python zmq library on Windows would cause Access violation sometimes.
 # - cmd: pip install zmq
 - cmd: echo set(VCPKG_BUILD_TYPE release) >> C:\tools\vcpkg\triplets\%PLATFORM%-windows-static.cmake
 - cmd: vcpkg remove --outdated --recurse
 - cmd: vcpkg install --triplet %PLATFORM%-windows-static %PACKAGES% > NUL
+# The libprotobuf-lite.lib has duplicate symbols to libprotobuf.lib and breaks bitcoin-qt and test_bitcoin-qt when ENABLE_BIP70 is set.
+- ps: |
+      if(Test-Path -Path ($env:LIBPROTOBUFLITE_PATH)) {
+         Write-Host "Moving libprotobuf-lite library to manual link directory.";
+         Move-Item -Path $env:LIBPROTOBUFLITE_PATH -Destination $env:LIBPROTOBUFLITE_DESTINATION_PATH
+      }
 before_build:
 - ps:  clcache -M 536870912
-- cmd: python build_msvc\msvc-autogen.py
-- ps:  $files = (Get-ChildItem -Recurse | where {$_.extension -eq ".vcxproj"}).FullName
-- ps:  for (${i} = 0; ${i} -lt ${files}.length; ${i}++) {
-           ${content} = (Get-Content ${files}[${i}]);
-           ${content} = ${content}.Replace("</RuntimeLibrary>", "</RuntimeLibrary><DebugInformationFormat>None</DebugInformationFormat>");
-           ${content} = ${content}.Replace("<WholeProgramOptimization>true", "<WholeProgramOptimization>false");
-           Set-Content ${files}[${i}] ${content};
-       }
```

Comment:
The WholeProgramOptimization is to make sure that the generated cache is not too big to reduce cache save/load time. I remember that the cache could be over 1GB without this setting so that we can't store it with free appveyor plan.

Reply:
The WholeProgramOptimization is off by default and I removed the cases where it was being set in individual project files.